zeppelin-interpreter-integration/pom.xml

```diff
@@ -203,4 +172,105 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>hadoop2</id>
+
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+
+      <properties>
+        <hadoop.version>${hadoop2.7.version}</hadoop.version>
+        <curator.version>2.7.1</curator.version>
```

Comment:
I do not see any use of `curator.version`, this property should be removed.

Reply:
fixed